images/linux/scripts/helpers/etc-environment.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+################################################################################
+##  File:  etc-environment.sh
+##  Desc:  Helper functions for source and modify /etc/environment
+################################################################################
+
+# NB: sed expression use '%' as a delimiter in order to simplify handling
+#     values containg slashes (i.e. directory path)
+#     The values containing '%' will break the functions
+
+function getEtcEnvironmentVar {
+    var_name="$1"
+    # remove `var_name=` and possible quotes from the line
+    grep "^${var_name}=" /etc/environment |sed -E "s%^${var_name}=\"?([^\"]+)\"?.*$%\1%"
+}
+
+function addEtcEnvironmentVar {
+    var_name="$1"
+    var_value="$2"
+
+    echo "$var_name=\"$var_value\"" | sudo tee -a /etc/environment
+}
+
+function replaceEtcEnvironmentVar {
+    var_name="$1"
+    var_value="$2"
+
+    # modify /etc/environemnt in place by replacing a string that begins with var_name
+    sudo sed -ie "s%^${var_name}=.*$%${var_name}=\"${var_value}\"%" /etc/environment
+}
+
+function setEtcEnvironmentVar {
```

Comment:
`setEtcEnvironmentVariable` - let's use the full `variable` word for readability

Reply:
The same for other functions please